packages/api-request-builder/src/default-params.js

```diff
@@ -77,16 +76,13 @@ export function setDefaultParams() {
   }
 
   if (this.features.includes(features.search)) {
-    this.params.staged = getDefaultSearchParams().staged
     this.params.pagination = getDefaultSearchParams().pagination
     this.params.search = getDefaultSearchParams().search
   }
 
   if (this.features.includes(features.queryLocation))
     this.params.location = getDefaultQueryParams().location
 
-  if (this.features.includes(features.projection)) this.params.staged = true
```

Comment:
Just so I follow:

1. Before `staged` was `true` by default
2. Then we check if we search to set it to `getDefaultSearchParams().staged`
   - Which is also `true` as per [ref](https://github.com/commercetools/nodejs/blob/master/packages/api-request-builder/src/default-params.js#L44)
3. Then we check if we're looking at the projection and set it to `true`

I am just wondering if there is a deliberate product need buried here. From old days.

Reply:
Yeah this is how I perceived the workflow too. No idea about any possible old need or not though.

@Oehmi ?